src/controls/DatetimeControl.tsx

```diff
@@ -0,0 +1,35 @@
+import { ControlProps } from "@jsonforms/core"
+import { withJsonFormsControlProps } from "@jsonforms/react"
+import { DatePicker, Form } from "antd"
+import dateFnsGenerateConfig from "rc-picker/lib/generate/dateFns"
+
+interface DatetimeControlProps extends ControlProps {
+  data: string
+  handleChange(path: string, value: Date | null): void
+  path: string
+}
+
+const MyDatePicker = DatePicker.generatePicker<Date>(dateFnsGenerateConfig)
+
+export function DatetimeControl({
+  data,
+  handleChange,
+  path,
+  label,
+  visible,
+  id,
+  required,
+}: DatetimeControlProps) {
+  const dateFormat = "MM/DD/YYYY"
```

Comment:
Can you make this configurable via UISchema or jsonschema / use browser-detected locale as a default?